_includes/author.html

```diff
@@ -1,71 +1,85 @@
 {%- if author -%}
-<div class="author-block">
-  <div class="author-photo">
-    {%- if author.gravatar -%}
-    <img
-      class="author-thumb"
-      src="//2.gravatar.com/avatar/{{ author.gravatar }}?s=50"
-      alt="{{ author.display_name }}"
-      nopin="nopin"
-    />
-    {%- else -%}
-    <img
-      class="author-thumb"
-      src="{{ site.baseurl }}/assets/images/profile.png"
-      alt="Author's profile picture"
-      nopin="nopin"
-    />
-    {%- endif -%}
-  </div>
+  <div class="author-block">
+    <div class="author-photo">
+      {%- if author.gravatar -%}
+      <img
+        class="author-thumb"
+        src="//2.gravatar.com/avatar/{{ author.gravatar }}?s=50"
```

Comment:
should this be http:// ?

Reply:
I don't know, it was like that before

Reply:
Do you think I should change it, @rdormer?